README.md

```diff
@@ -81,20 +63,14 @@ To setup `Percy`, you will need a token that you can find on Percy's dashboard:
 - [**Storybook**](https://storybook.js.org/) - Tool for developing UI components in isolation with documentation.
 - [**Debug**](https://github.com/visionmedia/debug) - JS debugging utility that works both in node.js and browsers.
 
-## Changing stuff
-
-### Removing Emotion
-
-- Uninstall npm packages
+## License
 
-```sh
-$ npm uninstall @emotion/core @emotion/styled @emotion/babel-preset-css-prop
-```
+GPL-3.0
```

Comment:
Change to apache, also change license file and package. Json license field

Reply:
ok